woocommerce/woocommerce-ios
# Enabled Products tab & M1 editing features by default

Fixes #2033 

## Changes
- Hide the products switch in the beta features settings screen.
- Removed all the unused code in `MainTabBarController` and refactoring of some methods
- Updated tests 
- New description inside the "top banner" in Product list

## Testing
- Verify that there is no mention of products in the Beta Features section in the Settings page.
- Verify that the products tab is displayed by default.
- Verify that product editing features is enabled for simple products only.
- M2 features are not displayed in the release version of the app.

Update release notes:
